fix(content): HR Cumulative Playthrough Edits (#7517)
endless-sky/endless-sky: data/hai/hai reveal 1 intro.txt

		log "Minor People" "Maduena Kabiru" `A senator from Bounty in the Delta Velorum system. He is somewhat paranoid and insistent upon the Navy taking great steps to better explore the galaxy beyond human space to search for possible threats.`
		conversation
			`As you walk through the spaceport, you can't help but notice that coverage on the events of the Pug invasion still continues to dominate headlines, even though it's been weeks now.`
			`	Looking around, you see that experts and officials have given way to talking heads and theorists. Some of them are saying what you yourself suspect: that the Pug invasion was a ploy to cause humanity to band together, learn to resolve differences without conflict, and ensure the sorts of extremism that led to the events precipitating the war would not be tolerated.`
			`	Others...`

> 			`	One particular paranoid fringe-world politician dominates the air on the channel you're watching here. The Senator Maduena Kabiru from Bounty demands that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people. While one could definitely argue in favor of his point, it feels rather cheap coming from a senator whose system is a mere two jumps from a brand new Navy base.`

Reviewer: ```suggestion
			`	One particular paranoid fringe-world politician dominates the air on the channel you're watching here. Senator Maduena Kabiru from Bounty demands that the edges of human space be better understood, that threats and even potential threats must clearly exist beyond what is already known, and that it is the responsibility of the government to find out and inform the people. While one could definitely argue in favor of his point, it feels rather cheap coming from a senator whose system is a mere two jumps from a brand new Navy base.`
```

No need for article on the formal title.

			`	As you are making the walk you hear questions posed and rumors stated. Occasionally you hear words like "honorable" and "reasonable" thrown in amongst the babble.`
			label main
			`	You arrive at a very comfortable office and are received by a mature-looking Hai with the appropriate regalia of a diplomat. She holds out a paw to greet you and says, "My name is Sayari. I was the envoy to the human government a century ago."`

> 			`	The present media storm is all the evidence you need to know that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then they're probably still trying to work out a suppression strategy and have been too busy to send an envoy quicker than your own trip took.`

Reviewer: ```suggestion
			`	The present media storm is all the evidence needed to make it clear that the Republic has never mentioned the Hai, nor publicly acknowledged receiving an envoy from them. If they're in on the secret, then they're probably still trying to work out a suppression strategy and have been too busy to send an envoy quicker than your own trip took.`
```

Moves it further back from assuming character/player knowledge, instead just presenting the knowledge as obvious.

			`	In an oddly human gesture that feels very deliberate, he places one of his furry hands comfortingly on your shoulder, "I'm sure you did all you could."`
				goto end
			label "saved geocoris"
			`	"Your rescue of our kidnapped people is deeply appreciated also. Lucky for you, the merchant who owned the Geocoris has already filed the insurance claim, so you get to keep the ship as a trophy.`

> 			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go though, so if you have any business on-world here, please take your time."`

Reviewer: ```suggestion
			`	"I do need to do a full security scan of it to complete all the incident paperwork before I let it go, though, so if you have any business on-world here, please take your time."`
```

Even in speech, I literally (not ironically literally either, quite literally literally) cannot say this sentence without the pause. Probably because of the rhyme.
